packages/govern-create/buidler.config.js

```diff
@@ -0,0 +1,135 @@
+usePlugin('solidity-coverage')
+usePlugin('@nomiclabs/buidler-ethers')
+usePlugin('@nomiclabs/buidler-etherscan')
+usePlugin('@nomiclabs/buidler-waffle')
+
+require('dotenv').config({ path: '../../.env' })
+
+const { readFileSync, writeFileSync } = require('fs')
+const {
+  uniqueNamesGenerator,
+  adjectives,
+  colors,
+  animals,
+} = require('unique-names-generator')
+
+const FACTORY_CACHE_NAME = 'govern-factory-rinkeby'
+const REGISTER_EVENT_NAME = 'Registered'
+const REGISTRY_EVENTS_ABI = [
+  'event Registered(address indexed dao, address queue, address indexed registrant, string name)',
+  'event SetMetadata(address indexed dao, bytes metadata)',
+]
+
+task('accounts', 'Prints the list of accounts', async () => {
```

Comment:
I think this file's a bit crowded. What do you think about:
- Moving the utility functions like `print` and `format` to an `lib/utils.js` file? We've standardized on this across all newer repos.
- Moving tasks to a `scripts` folder.

Let me know what do you think! I can also do these.

Reply:
Let's do later, pls create issue